edge-modules/TemperatureFilter/TemperatureFilter.csproj

```diff
@@ -19,6 +19,10 @@
     <OutputPath>bin\CodeCoverage</OutputPath>
     <DefineConstants>DEBUG;TRACE</DefineConstants>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <Content Include="docker*/**/*.*" CopyToPublishDirectory="Always" />
```

Comment:
Is *.* correct? Is this not a pattern?

Reply:
I tested and it is used in other existing project except this one.